mage/query-modules/cpp/community_detection.md

```diff
@@ -0,0 +1,142 @@
+---
+id: community-detection
+title: community_detection
+sidebar_label: community_detection
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const Highlight = ({children, color}) => (
+  <span
+    style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+    {children}
+  </span>
+);
+
+[![docs-source](https://img.shields.io/badge/source-community_detection_online-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/blob/main/cpp/community_detection_module/community_detection_module.cpp)
+
+## Abstract
+
+This query module makes available the [Louvain method](https://arxiv.org/abs/1305.2006)
+<sup>[1]</sup> for community detection, using the [Grappolo](https://github.com/Exa-Graph/grappolo)
```

Comment:
```suggestion
This query module makes available the [Louvain method](https://arxiv.org/abs/1305.2006) <sup>[1]</sup> for community detection, using the [Grappolo](https://github.com/Exa-Graph/grappolo)
```

Reply:
The formatting was weird before removing this newline

Reply:
Thanks!